src/Frontend/Core.hs

```diff
@@ -487,9 +487,10 @@ pageFrame frame = do
         link_ [rel_ "stylesheet", href_ $ P.TopStatic "css/all.css"]
 
         -- | disable the meta tag for admins, since admin pages are not working on mobile devices.
-        case frame ^? frameUser . userRole of
-            Just Admin -> meta_ [name_ "viewport", content_ "width=1024"]
-            _          -> meta_ [name_ "viewport", content_ "width=device-width, initial-scale=1"]
+        let viewport_content
+                | allOf frameUser isAdmin frame = "width=1024"
```

Comment:
should this be `anyOf`?   an admin who is also moderator can see the admin pages, so she shouldn't be switched to mobile.  see comment above.